onetl/db/db_reader/db_reader.py

```diff
@@ -545,8 +591,8 @@ def run(self) -> DataFrame:
         self.connection.check()
 
         helper: StrategyHelper
-        if self.hwm_column:
-            helper = HWMStrategyHelper(reader=self, hwm_column=self.hwm_column, hwm_expression=self.hwm_expression)
+        if self.hwm is not None:
```

Comment:
```suggestion
        if self.hwm:
```
В целом, все такие вхождения `hwm is not None` можно заменить на просто `hwm`